singularity-eos/eos/singularity_eos.cpp

```diff
@@ -20,6 +20,56 @@
 
 using namespace singularity;
 
+// TODO: Replace these with the new Modify method in EOSBuilder.
+// NOTE: The new EOSBuilder machinery will likely be slower than these.
+template <typename T>
+EOS applyShiftAndScale(T &&eos, bool scaled, bool shifted, Real scale, Real shift) {
```

Comment:
@dholladay00 if we end up keeping the rewrite, we should convert `singularity_eos.cpp` to use the new builder code at some point. But for now, I left the older `applyXY*` stuff, as it works without intervention. It may also be faster to compile on the full variant.